platform/mbed_wait_api.h

```diff
@@ -41,59 +41,26 @@ extern "C" {
  * @code
  * #include "mbed.h"
  *
- * DigitalOut heartbeat(LED1);
+ * // Blinking rate in milliseconds
+ * #define BLINKING_RATE_MS   500
+ * DigitalOut  led(LED2);
+ * InterruptIn button(SW2);
+ *
+ * void blink_led() {
+ *     led = 1;
+ *     wait_us(BLINKING_RATE_MS * 1000);
```

Comment:
`wait_us` doesn't exist anymore. Could you grep the sources for any occurance of this function (and wait in general) even in comments. Also have a look at all the stand alone examples and sinpets in the docs (there's a separate repository for them).

Reply:
Nevermind, I just noticed it's example for wait* function.

Reply:
I was about to ask why the function no longer exists.